### ISSUE TITLE
Skip Intro & Exit music (& episode recaps?)

### ISSUE THREAD

**Issue by chellygel** [chellygel/AnimeKetchup#9]
This issue should be reserved for fixing during API usage as i feel the user input command line version of Anime Ketchup is getting a bit wordy and long.

Some users wish to save time by skipping intro and exit music. These can last anywhere from 30 seconds to 1 minute and 30 seconds. Allow a user to factor in how much time is saved if they skipped these.

Future Considerations:
1. We should track the length of OP (intro) and ED (endings) within our system
1. Not all shows have episode recaps, but we could consider that as well

**Comment by leepatrickworrall** [comment]
An additional future consideration too to add onto the rest. Yeah, not all shows have episode recaps, but also they do have cold opens. Say if an episode is a multipart episode. That needs to be factored in.
